src/wwwroot/controllers/MainCtrl.js

```diff
@@ -17,6 +17,8 @@
   ];
 
   function MainCtrl($rootScope, $scope, $window, $location, auth, $log, ModalService, $route) {
+    $rootScope.freeTrialEndDate = auth.getFreeTrialEndDate;
```

Comment:
What do you think about renaming `$rootScope.freeTrialEndDate` as `$rootScope.getFreeTrialEndDate`? I feel that without the verb it is not clear that it is a method, it looks like an accessor.